docs/quickstart-traffic-splitting.md

```diff
@@ -1,35 +1,24 @@
-# Traffic splitting quickstart
+# Traffic splitting quick start
 
-This quickstart guide enumerates the steps needed for using ModelMesh with
-traffic splitting. Please, note that this results in a demo/experimental setup.
+This `odh-model-controller` quick start demonstrates how to use the ModelMesh framework with traffic splitting. 
 
 ## Prerequisites
 
-Since `odh-model-controller` is a companion controller
-for [modelmesh-serving](https://github.com/opendatahub-io/modelmesh-serving),
-you must first install it. Please, follow
-[ModelMesh serving quickstart's installation section](https://github.com/opendatahub-io/modelmesh-serving/blob/main/docs/quickstart.md#1-install-modelmesh-serving).
+- Install ModelMesh as described in the [ModelMesh serving quickstart's installation section](https://github.com/opendatahub-io/modelmesh-serving/blob/main/docs/quickstart.md#1-install-modelmesh-serving). The `odh-model-controller` is a companion controller for [modelmesh-serving](https://github.com/opendatahub-io/modelmesh-serving).
 
-## Install a Service Mesh
+- Install Service Mesh depending on your OpenShift or Kubernetes system.
 
-If you are using OpenShift, you can use
-the [deploy.ossm.sh](../scripts/deploy.ossm.sh) script to quickly install
-OpenShift Service Mesh (OSSM). If you are using ROSA, there are a few lines in
-the script that you will need to uncomment.
+  - On OpenShift, use the [deploy.ossm.sh](../scripts/deploy.ossm.sh) script to quickly install OpenShift Service Mesh (OSSM). On Red Hat OpenShift for AWS, you must uncomment a few lines in the `deploy.ossm.sh` script before you run it.
 
-If you are using another Kubernetes flavor (or also OpenShift), you can
-use [Istio](https://istio.io/). Use the installation method that better suits
-your needs. You must install `istiod` and `istio-ingressgateway` components. If
-you are
-[installing with `istioctl`](https://istio.io/latest/docs/setup/install/istioctl/),
-installing with the command `istioctl install` will provide a setup with these
-components.
+  - On another type of OpenShift (or Kubernetes), use [Istio](https://istio.io/) to install OSSM. You must also install the required `istiod` and `istio-ingressgateway` components. For example, if you [install with `istioctl`](https://istio.io/latest/docs/setup/install/istioctl/), use the `istioctl install` command which also installs `istiod` and `istio-ingressgateway`.
```

Comment:
Istio is the upstream project of OSSM. So, we cannot say that Istio is used to install OSSM. We just install Istio and that's it. Istio works in (almost?) any Kubernetes system (including OpenShift), while OSSM works only in OpenShift.  The `istiod` and `istio-ingressgateway` components are part of Istio and can be installed selectively.

Additionally, I just remembered that this repository is OpenShift specific and not meant for any other Kubernetes system.

I tried to fix this second bullet to denote all this, but I think my suggestion needs you refinement:

```suggestion
  - Alternatively, use [Istio](https://istio.io/). You must install the required `istiod` and `istio-ingressgateway` components. For example, if you [install with `istioctl`](https://istio.io/latest/docs/setup/install/istioctl/), using the `istioctl install` command includes the required `istiod` and `istio-ingressgateway` components.
```